<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>txmparameters</class>
 <widget class="QWidget" name="txmparameters">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>898</width>
    <height>821</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="atBeginButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="jle_resources.qrc">
           <normaloff>:/icons/Resources/pic/Button_ShiftUpUp.png</normaloff>:/icons/Resources/pic/Button_ShiftUpUp.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="oneBackButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="jle_resources.qrc">
           <normaloff>:/icons/Resources/pic/Button_ShiftUp.png</normaloff>:/icons/Resources/pic/Button_ShiftUp.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="oneForewardButton">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="jle_resources.qrc">
           <normaloff>:/icons/Resources/pic/Button_ShiftDown.png</normaloff>:/icons/Resources/pic/Button_ShiftDown.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="atEndButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="jle_resources.qrc">
           <normaloff>:/icons/Resources/pic/Button_ShiftDownDown.png</normaloff>:/icons/Resources/pic/Button_ShiftDownDown.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableView" name="jobListTableView">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="gbWaveform">
         <property name="title">
          <string>Waveform</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Name</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="cbTXMWaveformName">
            <item>
             <property name="text">
              <string>DefaultRect</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>DefaultSine</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>DefaultTriangle</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>DefaultRamp</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="5" column="0">
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Max Amplitude [A]</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="sspinTXMMaxAmpl"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Base Frequency [Hz]</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QComboBox" name="cbTXMBaseFreq">
            <item>
             <property name="text">
              <string>16384</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>8192</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>4096</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2048</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1024</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>512</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>256</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>128</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>64</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>32</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>16</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>8</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>4</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>0.5</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>0.25</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>0.125</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>0.0625</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="Line" name="line_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="Line" name="line_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="Line" name="line_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="Line" name="line_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="Line" name="line_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="pbTXMWaveformChangeAll">
            <property name="text">
             <string>Change in All Jobs</string>
            </property>
            <property name="icon">
             <iconset resource="jle_resources.qrc">
              <normaloff>:/icons/Resources/pic/Button_OK.png</normaloff>:/icons/Resources/pic/Button_OK.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>45</width>
              <height>45</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QPushButton" name="pbTXMAmplChangeAll">
            <property name="text">
             <string>Change in All Jobs</string>
            </property>
            <property name="icon">
             <iconset resource="jle_resources.qrc">
              <normaloff>:/icons/Resources/pic/Button_OK.png</normaloff>:/icons/Resources/pic/Button_OK.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>45</width>
              <height>45</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gbSequence">
         <property name="title">
          <string>Sequence</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QGridLayout" name="gridLayout_8">
            <item row="0" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Number of Stacks</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="pbTXMNumStacks">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Dipole TX Time [s]</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="pbTXMDipTXTime">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_6">
            <property name="title">
             <string>Dipoles</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <layout class="QGridLayout" name="gridLayout_9">
               <item row="0" column="0">
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>Number of Dipoles</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QComboBox" name="cbTXMNumDipoles">
                 <item>
                  <property name="text">
                   <string>8</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>6</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>4</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>2</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>1</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>Polarisation Dipole 1</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="1">
                <widget class="QDoubleSpinBox" name="sspinTXMPolDip2"/>
               </item>
               <item row="3" column="1">
                <widget class="QDoubleSpinBox" name="sspinTXMPolDip1"/>
               </item>
               <item row="5" column="1">
                <widget class="QDoubleSpinBox" name="sspinTXMPolDip3"/>
               </item>
               <item row="6" column="1">
                <widget class="QDoubleSpinBox" name="sspinTXMPolDip4"/>
               </item>
               <item row="4" column="0">
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>Polarisation Dipole 2</string>
                 </property>
                </widget>
               </item>
               <item row="5" column="0">
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>Polarisation Dipole 3</string>
                 </property>
                </widget>
               </item>
               <item row="6" column="0">
                <widget class="QLabel" name="label_10">
                 <property name="text">
                  <string>Polarisation Dipole 4</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QFrame" name="frame_3">
                 <property name="frameShape">
                  <enum>QFrame::Panel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QFrame" name="frame_4">
                 <property name="frameShape">
                  <enum>QFrame::Panel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="Line" name="line_7">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="Line" name="line_8">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item row="3" column="2">
                <widget class="QLabel" name="label_11">
                 <property name="text">
                  <string>Polarisation Dipole 5</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="3">
                <widget class="QDoubleSpinBox" name="sspinTXMPolDip5"/>
               </item>
               <item row="4" column="2">
                <widget class="QLabel" name="label_12">
                 <property name="text">
                  <string>Polarisation Dipole 6</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="3">
                <widget class="QDoubleSpinBox" name="sspinTXMPolDip6"/>
               </item>
               <item row="5" column="2">
                <widget class="QLabel" name="label_13">
                 <property name="text">
                  <string>Polarisation Dipole 7</string>
                 </property>
                </widget>
               </item>
               <item row="5" column="3">
                <widget class="QDoubleSpinBox" name="sspinTXMPolDip7"/>
               </item>
               <item row="6" column="2">
                <widget class="QLabel" name="label_14">
                 <property name="text">
                  <string>Polarisation Dipole 8</string>
                 </property>
                </widget>
               </item>
               <item row="6" column="3">
                <widget class="QDoubleSpinBox" name="sspinTXMPolDip8"/>
               </item>
               <item row="1" column="2">
                <widget class="Line" name="line_9">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item row="1" column="3">
                <widget class="Line" name="line_10">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QPushButton" name="pbTXMDipoleChangeAll">
               <property name="text">
                <string>Change Dipole Settings in All Jobs</string>
               </property>
               <property name="icon">
                <iconset resource="jle_resources.qrc">
                 <normaloff>:/icons/Resources/pic/Button_OK.png</normaloff>:/icons/Resources/pic/Button_OK.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>45</width>
                 <height>45</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="jle_resources.qrc"/>
 </resources>
 <connections/>
</ui>
